src/commands/verify.ts

```diff
@@ -0,0 +1,36 @@
+import { execSync } from "child_process"
+import { readFileSync } from "fs"
+import { existsSync } from "fs"
+import { writeFileSync } from "fs"
+import { resolve } from "path"
+import { PackageFormat } from "../api/PackageFormat"
+import { getPathToPackages } from "../fileHelpers"
```

Comment:
nit adding `util` to the path fixes the build.

Reply:
```suggestion
import { getPathToPackages } from "../util/fileHelpers"
```